jenkins/ci.suse.de/pipelines/cloud-ccp-full-periodic-airship.Jenkinsfile

```diff
@@ -0,0 +1,96 @@
+pipeline {
+  options {
+    timestamps()
+    parallelsAlwaysFailFast()
+    skipDefaultCheckout()
+  }
+
+  agent {
+    node {
+      label "cloud-ccp-ci"
+      customWorkspace "${JOB_NAME}-${BUILD_NUMBER}"
+    }
+  }
+
+  environment {
+    DEPLOYMENT_MECHANISM = "openstack"
+    OS_CLOUD = "engcloud-cloud-ci"
+    KEYNAME = "engcloud-cloud-ci"
+    SOCOK8S_ENVNAME = "ccp-airship-${env.BUILD_NUMBER}"
+    SOCOK8S_DEVELOPER_MODE = "True"
+    DELETE_ANYWAY = "YES"
+    ANSIBLE_STDOUT_CALLBACK = "yaml"
+  }
+
+  stages {
+    stage('Show environment information') {
+      steps {
+        sh 'printenv'
+      }
+    }
+
+    stage('Clone socok8s repo') {
+      steps {
+        sh "git clone ${ccp_repo} -b ${ccp_branch} ."
+      }
+    }
+
+    stage('Setup network') {
+      steps {
+        sh "./run.sh deploy_network"
+      }
+    }
+    stage('Setup hosts') {
+      parallel {
+        stage('Setup SES') {
+          steps {
+            sh "./run.sh deploy_ses"
+          }
+        }
+        stage('Setup CaaSP') {
+          steps {
+            sh "./run.sh deploy_caasp"
+          }
+        }
+        stage('Setup Deployer') {
+          steps {
+            sh "./run.sh deploy_ccp_deployer"
+          }
+        }
+      }
+    }
+    stage('Enroll CaaSP workers') {
+      steps {
+        sh "./run.sh enroll_caasp_workers"
+      }
+    }
+
+    stage('Deploy Airship') {
+      steps {
+        sh('''
+          USER=${USER:-root}  # why isn't this set?
+          ./run.sh setup_airship
```

Comment:
does this step deploy also openstack helm or is this separate? I think what w eneed to do is deploy airship, create the site manifest and then let airship deploy that site manifest which should deploy the openstack helm charts

Reply:
Yes, this step does everything needed to deploy openstack using airship